SimPEG/Survey.py

```diff
@@ -149,8 +149,8 @@ def vnD(self):
         return np.array([rx.nD for rx in self.rxList])
 
 
-class Data(object):
-    """Fancy data storage by Src and Rx"""
+class BaseData(object):
```

Comment:
I agree @lheagy that we should use `properties` for this down the road. But for this work surprisingly well.